cypress/support/step-definitions/validations-steps.js

```diff
@@ -0,0 +1,77 @@
+import { getDataElementByValueNoIframe, commonDataElementInputPreviewNoIframe } from '../../locators';
+import { asyncWaitForIcon } from '../helper';
+import { errorMessageNoIframe, inputValidationNoIframe } from '../../locators/validations';
+
+const FIRST_ELEMENT = '0';

```

Comment:
make the values as `Integer`s instead of string

Reply:
fixed